src/main/java/baseball/Domain/CompareNumber.java

```diff
@@ -0,0 +1,29 @@
+package baseball.Domain;
+
+import java.util.List;
+
+public class CompareNumber {
+
+
+
+
+	public int Strike(List<Integer> playerNum , List<Integer> comNum){
```

Comment:
- 줄바꿈, 공백도 컨벤션입니다
```suggestion

	public int Strike(List<Integer> playerNum, List<Integer> comNum){
```

지금은 어플리케이션이 작으니 상관없지만, 
정언짱의 협업자가 20줄을 띄우고, `,`를 제멋대로 찍는다고 상상해보세요 ㅋㅋ

Reply:
* 수정하였습니다!

Reply:
* 수정했습니다 앞으로 컨벤션을 잘 지켜 코딩을 하도록 하겠습니다. 나중에 유지보수 할 때도 불편하겠군요 !